css/buttons.css

```diff
@@ -0,0 +1,80 @@
+.btn {
```

Comment:
this is converted from scss, hence why the indentation

Reply:
Does the SCSS conversion process not have any configuration or are there no linting tools that can set/fix this to our usual standard?

If not, it's 80 lines of a file. Manually do it if you have to.